app/controllers/settings.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+const { Controller, get, inject, computed } = Ember;
+
+export default Controller.extend({
+  session: inject.service(),
+  isAuthenticated: computed.oneWay('session.isAuthenticated'),
+
+  actions: {
+    signOut() {
```

Comment:
I tried creating the action as a route-action but it meant that I have to have a session both in my route and my controller. Does this still make sense?

Anyways I guess this can bubble all the way up to application. However this is the only place in the app currently where you can sign out. 

In a big application would this still be the best idea? 
Eventually you will reach a place where you have too many actions on your application route